### ISSUE TITLE
508-defect-1 ❗ Launchblocker [SCREENREADER, COGNITION]: Language of page MUST be set to ES

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#18626]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Josh_

## User Story or Problem Statement

As a screen reader user, I want to hear the content of a Spanish page pronounced accurately.

## Details

The page's language is set to `en` (English) instead of `es` (Spanish). Changing the attribute value to `es` will help:
- people who use screen readers or other technologies that convert text into synthetic speech;
- people who find it difficult to read written material with fluency and accuracy, such as recognizing characters and alphabets or decoding words;
- people with certain cognitive, language and learning disabilities who use text-to-speech software
- people who rely on captions for synchronized media.

## Acceptance Criteria

- [ ] The page's `<html lang="en">` is changed to `<html lang="es">`

## Steps to Recreate

1. Enter `https://www.va.gov/coronavirus-veteran-frequently-asked-questions-esp/` in browser
2. Confirm the page's `lang` attribute is equal to `en` instead of `es`

## Proposed Solution 

Change the page's `<html lang="en">` to `<html lang="es">`

## WCAG or Vendor Guidance 

* [WCAG 2.0 A: Success Criterion 3.1.1: Language of Page](https://www.w3.org/WAI/WCAG21/Understanding/language-of-page.html)

## Screenshots or Trace Logs
<img width="1194" alt="Screen Shot 2021-01-15 at 11 17 32 AM" src="https://user-images.githubusercontent.com/14154792/104751437-6121a980-5723-11eb-9729-50cc947bd2ff.png">

**Comment by joshkimux** [comment]
In discussion with @1Copenut , @youngfreezyVA , and @noahgelman we will be closing out this ticket as a short-term fix for a larger, more complex problem. In order to meet Section 508 compliance, we've opted to add in additional acceptance criteria for [ticket 18698](https://github.com/department-of-veterans-affairs/va.gov-team/issues/18698).

Some additional notes:
- In Chrome, the `alert` on this page will _always_ be read out on page load in the user's screen reader's language preference. As such, the hypothesis of changing the `lang` of the page to `es` will _not_ be able to fix the scenario where a screen reader user has their preference set to English, but still hears the alert on page load in Spanish.
- We've decided to continue with the JS insertion of the `lang="es"` to a `div` wrapping most of the `main` due to constraints in regards to editing the `header` and `footer` to have a `lang="en"` to make up for the lack of a Spanish `header` and `footer`.
- There may be issues regarding how the metadata of the page is interpreted if the language of the page is set to `en` as opposed to `en`. 
- Ideally, in the future, we hope to recommend creating a Spanish version `header` and `footer` of the VA.gov website so we can ultimately switch the page's `lang` to `es` as opposed to `en`. This also would resolve the potential issue of the `lang` not updating to `es` should the user not have JS running.
